backend/src/main/java/com/crm/dto/response/CustomerResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.crm.dto.response;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Objects;
+
+@Getter
+@Setter
+@Builder
+public class CustomerResponse {
+    private Long id;
+    private String name;
+    private String surname;
+    private String phone;
+    private String address;
+
```

Comment:
Is it generated method? Consider using annotation @EqualsAndHashCode

Reply:
I decided to use `@EqualsAndHashCode`, thanks